index.html

```diff
@@ -3,31 +3,32 @@
   <head>
     <meta charset="utf-8">
     <meta name="description" content="Hello Studio Website.">
+    <link rel="stylesheet" href="style/main.css">
     <title>Hello Studio</title>
   </head>
   <body>
-    <header>
-      <h1>
-        <span>the time is</span>
-        <span>now</span>
+    <header class="header">
+      <h1 class="header__heading">
+        <span class="header__heading1">the time is</span>
+        <span class="header__heading2">now</span>
       </h1>
-      <nav>
-        <ul>
-          <li><a href="#intro">intro</a></li>
-          <li><a href="#gallery">gallery</a></li>
-          <li><a href="#services">services</a></li>
-          <li><a href="#contact">contact</a></li>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#intro" class="nav__link">intro</a></li>
+          <li class="nav__item"><a href="#gallery" class="nav__link">gallery</a></li>
+          <li class="nav__item"><a href="#services" class="nav__link">services</a></li>
+          <li class="nav__item"><a href="#contact" class="nav__link">contact</a></li>
         </ul>
       </nav>
-      <section>
-        <ul>
-          <li><a href="https://pinterest.com">Pinterest<img src="assets/pinterest.svg" aria-hidden="true"></a></li>
-          <li><a href="https://facebook.com">Facebook<img src="assets/facebook.svg" aria-hidden="true"></a></li>
-          <li><a href="https://twitter.com">Twitter<img src="assets/twitter.svg" aria-hidden="true"></a></li>
+      <section class="media">
```

Comment:
Może `social-media`? Musiałam spojrzeć na linki żeby ogarnąć który to element :)